test/test_playbooks/tasks/repository.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "Create katello repository"
+  vars:
+    - organization: "Test Organization"
+    - product_name: "product1"
```

Comment:
For consistency these might be called `repository_{organization,product}`

Reply:
I was going for simplicity of using the same data in multiple places.

Reply:
I'd agree with @gregswift here. I think any top-level attributes can be thought of as repository_<attribute> without being explicit.

Reply:
Updated above comment.

Reply:
As long, as there cannot arise conflicts, i'm fine.